CMakeModules/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 
 set(MODULE_FILES
   Findmc_rtc_3rd_party_geos.cmake
-  Findmc_rtc_3rd_party_hpp-spline.cmake
+  Findmc_rtc_3rd_party_ndcurves.cmake
```

Comment:
I don't think we need a specific module for this, any recent version of ndcurves should have a valid CMake config file